### DISCUSSION TITLE
Checksum mismatch between different download locations v2.4.9.4

### DISCUSSION THREAD

**Discussion by 125IV543** [OpenIDC/mod_auth_openidc#676] [Q&A] [answered]
Hi,

why checksums are inconsistent between the two download locations?

https://github.com/zmartzone/mod_auth_openidc/archive/v2.4.9.4.tar.gz
sha512: cc4850cf88e5920fd944f5865f2bf0072f12d26a7f5aad38f378412dec01a9698c899616320a584a6e6d81f5dd50aaa1f5598cdc7cb50df6215dc516fa507d4e

https://github.com/zmartzone/mod_auth_openidc/releases/download/v2.4.9.4/mod_auth_openidc-2.4.9.4.tar.gz
755f3343f87cd1b05d16e60afc86b68a79657adf793863215fba1bce6a12b8b7b18f60fba5041f82601d3ab2d526287ff3f88fef33673e84456d0423a90b1977

**Comment by zandbelt** [comment, marked as answer]
one are github provided sources - a copy of the repo - the other are the net sources needed to build the module as used for packaging for various distributions i.e. after running autogen.sh
